src/openloco/industry.cpp

```diff
@@ -113,7 +113,7 @@ namespace openloco
                         {
                             if (prng.rand_bool())
                             {
-                                map::map_pos randTile{ x + coord_t(prng.rand_next(-15, 16) * 32), y + coord_t(prng.rand_next(-15, 16) * 32) };
+                                map::map_pos randTile{ x + static_cast<coord_t>(prng.rand_next(-15, 16) * 32), y + static_cast<coord_t>(prng.rand_next(-15, 16) * 32) };
```

Comment:
the `+` is causing this to be promoted to an int. just do a static cast of the whole thing ` static_cast<coord_t>(x + prng.rand_next(-15, 16) * 32)`

Reply:
Thanks, I was getting frustrated with this!